ovos_plugin_manager/templates/agents.py

```
@@ -134,3 +141,601 @@ class MultimodalAdapter(ABC):
+            messages (List[AgentMessage]): Full list of messages.
+            session_id (str): Identifier for the session.
+            lang (str, optional): Language code.
+            units (str, optional): Unit system.
+
+        Returns:
+            Iterable[AgentMessage]: A stream of response messages.
+        """
+        yield self.continue_chat(messages, session_id, lang, units)
+
+    def get_response(self, utterance: str,
+                     image_content: List[str] = None,  # b64 encoded
```

Ruff (0.14.14)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)